### DISCUSSION TITLE
Instancing Text objects

### DISCUSSION THREAD

**Discussion by SrinivasPrabhu794** [pmndrs/drei#1388] [Q&A]
Hi,

I was wondering if there is a way to **instance Text objects** in a R3F scene similar to how we can render geometries with different positions/colors using instancedMesh in R3F?

In my case, I have instanced geometries, mostly planes in the **order of 10-50k**. Each of these planes has its corresponding label as well.

**Rendering 50k Text in R3F** could be heavy.  Let me know if anyone has tried the same?

Thanks
Srinivas

**Comment by amitsinghio** [comment]
Hi there, 

wondering about the same thing. Did you solve this problem?

best regards,
Amit
